deployments/compose/dbt/dbt-project/README.md

```diff
@@ -0,0 +1,15 @@
+Welcome to your new dbt project!
+
+### Using the starter project
+
+Try running the following commands:
+- dbt run
+- dbt test
```

Comment:
?
```shell
$ dbt run
$ dbt test
```

Reply:
Generated as part of the `dbt init`. Leaving it
Are you suggesting adding `dbt test` to `deployments/compose/README.md`?
I don't think it's needed

Reply:
I thought you wanted to use the markdown syntax that will look like shell